--- config.py	2018-03-02 22:21:34.119562000 -0600
+++ config-single.py	2018-03-02 22:13:20.544781000 -0600
@@ -1,10 +1,10 @@
 import sys
 
-# Doing this allows us to have a unified code in master main for both single/multiple test sceanrios
-WATCHDOG_PORT = [6666, 6666, 6666, 6666, 6666]
-# CLIENT_PORT = 8888
+
+WATCHDOG_PORT = [6666, 6667, 6668, 6669, 6670]
 CLIENT_PORT = [9001, 9002, 9003, 9004, 9005]
 SERVER_PORT = [8000, 8001, 8002, 8003, 8004]
+
 LOG_FILE = sys.stdout
 file_dict_dir = "dict/"
 LOG_DIR = "log/"
@@ -13,24 +13,23 @@
 NUM_SERVER = 5
 
 ADDR_PORT = {
-    "0": ('128.83.139.248', 8000, "server"),
-    "1": ('128.83.130.139', 8001, "server"),
-    "2": ('128.83.120.63', 8002, "server"),
-    "3": ('128.83.139.77', 8003, "server"),
-    "4": ('128.83.139.114', 8004, "server"),
-    "5": ('128.83.139.250', 9001, "client"),
-    "6": ('128.83.139.78', 9002, "client"),
-    "7": ('128.83.130.140', 9003, "client"),
-    "8": ('128.83.130.142', 9004, "client"),
-    "9": ("128.83.144.59", 9005, "client"),
+    "0": ('localhost', 8000, "server"),
+    "1": ('localhost', 8001, "server"),
+    "2": ('localhost', 8002, "server"),
+    "3": ('localhost', 8003, "server"),
+    "4": ('localhost', 8004, "server"),
+    "5": ('localhost', 9001, "client"),
+    "6": ('localhost', 9002, "client"),
+    "7": ('localhost', 9003, "client"),
+    "8": ('localhost', 9004, "client"),
+    "9": ("localhost", 9005, "client"),
 }
 
+
 SERVER_COUNT = 5
 CLIENT_COUNT = 5
-# WATCHDOG_IP_LIST = ["localhost"] * SERVER_COUNT
-# CLIENT_IP_LIST = ["localhost"] * CLIENT_COUNT
-WATCHDOG_IP_LIST = ['128.83.139.248','128.83.130.139','128.83.120.63', '128.83.139.77','128.83.139.114']
-CLIENT_IP_LIST = ['128.83.139.250','128.83.139.78','128.83.130.140','128.83.130.142','128.83.144.59']
+WATCHDOG_IP_LIST = ["localhost"] * SERVER_COUNT
+CLIENT_IP_LIST = ["localhost"] * CLIENT_COUNT
 
 GOSSIP_INTERVAL = 0.2
 # time period we resend the message to the server that hasn't ACK our message
